### ISSUE TITLE
Support TV type override (NTSC / JP roms on N64 PAL)

### ISSUE THREAD

**Issue by landsat77** [Polprzewodnikowy/N64FlashcartMenu#66]
### Menu version used

0.0.0.5 Alpha

### Description

Using PAL Fra RGB modded N64

Using no-intro roms, the same files are working well on my X7 and ED64+

Menu boot at 576i/50 or 480i/60 (ini settings)

Pal roms works fine, i found no issues yet.

USA / JAP Roms :

F-zero X => Works Fine
Star Fox => Works Fine
Resident Evil USA / JAP => black screen nothing happens even after a couple of minuts
Ogre Battle 64 USA / JAP => black screen nothing happens even after a couple of minuts
Wave Race USA / JAP => Boot but freeze some seconds later
Gex 3 => black screen nothing happens even after a couple of minuts
007 The World is not enough => boot but stuck on region lock screen
Paperboy => boot but stuck on region lock screen
Megaman 64 => => boot but stuck on region lock screen

I try to send rom over USB-C, same issue than loading over the menu.

**Comment by networkfusion** [comment]
@landsat77 , just for confirmation, please can you also add the exact command you sent to `scdeployer` and `scdeployer` version. Also, please confirm the output is the same with the very latest menu version from rolling release (pressing start should have a timestamp against 0.0.0.6) which is very useful for future revisions.

**Comment by networkfusion** [comment]
Also the output of `./sc64deployer.exe -info` to show the current FW version on your cart.

**Comment by landsat77** [comment]
sc64deployer.exe info
←[1mSC64 information and current state:←[0m
 Firmware version:    v2.17.0
 RTC datetime:        2022-03-03 14:03:43 +01:00
 Boot mode:           Bootloader -> Menu from SD card
 Save type:           None
 CIC seed:            Auto
 TV type:             Passthrough
 Bootloader switch:   Enabled
 ROM write:           Disabled
 ROM shadow:          Disabled
 ROM extended:        Disabled
 64DD mode:           Disabled
 64DD SD card mode:   Disabled
 64DD drive type:     Retail
 64DD disk state:     Ejected
 Button mode:         User
 Button state:        Not pressed
 LED blink:           Enabled
 IS-Viewer 64 offset: 0x00000000
 FPGA debug data:     Last PI address: 0x00000000
 MCU stack usage:     CIC: 96, RTC: 120, LED: 88, GVR: 312

**Comment by landsat77** [comment]
Using the last sc64menu.n64
![IMG_4507](https://github.com/Polprzewodnikowy/N64FlashcartMenu/assets/150617212/355deaa1-5128-4069-891b-788581d56550)

Loading **Paperboy (USA).z64** have the same result.

**Comment by Polprzewodnikowy** [comment]
By default `sc64deployer` has the same default behavior regarding "TV type" boot parameter as current version of N64FlashcartMenu. Currently game receives the same value as reported by the console. `sc64deployer` can override this value, to do this use `--tv [region]` parameter in the `upload` command to force TV type setting to the other value. There are more options available in the `sc64deployer`. Information about these parameters is bundled in the application and accessible via `--help` option. Note that different commands accept different parameters. Please look at the screenshot below:

![image](https://github.com/Polprzewodnikowy/N64FlashcartMenu/assets/3756990/65e00ae3-0f54-4be2-bf29-79b9d7497bc2)

To summarize, try running `sc64deployer.exe upload "game.z64" --tv ntsc` to boot NTSC game on PAL console. Also don't really know why you had to put `--port com9` argument, deployer should automatically detect and use correct SC64 device.

Support for overriding TV type and save parameters will be coming soon to the N64FlashcartMenu.

**Comment by landsat77** [comment]
using : 
D:\Summer 64>sc64deployer.exe upload "Paperboy (USA).z64" --tv ntsc
Uploading ROM [Paperboy (USA).z64]... ←[1;92mdone←[0m
ROM title: Paperboy (U) [!]
Save type set to [None]
Boot mode set to [Bootloader -> ROM]
TV type set to [NTSC]

Works fine now, tested with **Paperboy** and **Resident Evil 2**

**Comment by ottelo9** [comment]
I had the same problems with Altra64 and fixed the auto region detection (see [commit1](https://github.com/ottelo9/altra64/commit/81f04b95f1478b2a43f630e6eb6db23eb0a86de8) [commit2](https://github.com/ottelo9/altra64/commit/a05565b221417b2f67091da4883e331bfc810798). Maybe it will help. But I don't have time to deal with it at the moment.

**Comment by Polprzewodnikowy** [comment]
Partial support has been merged in https://github.com/Polprzewodnikowy/N64FlashcartMenu/pull/71
There's still no support to do manual override

**Comment by Polprzewodnikowy** [comment]
Manual TV type and save type has been implemented in this commit: https://github.com/Polprzewodnikowy/N64FlashcartMenu/commit/621e50cf7b78e5b427267ec59931ca461d9c8371
